src/lib/flexbox/api/base.ts

```diff
@@ -18,6 +18,13 @@ import {MediaMonitor} from '../../media-query/media-monitor';
 import {MediaQuerySubscriber} from '../../media-query/media-change';
 
 /**
+ * To avoid dependencies on full class Renderer or Renderer,
+ * define interface for the only method used...
+ */
+export interface StyleRenderer {
```

Comment:
Since the `BaseFxDirective` directive only uses the method `setElementStyle()` using a StyleRenderer interface clearly declares the intent of the reference.